File: se_test.rb
require 'selenium-webdriver'
require 'rspec/expectations'
include RSpec::Matchers

def setup
  @driver = Selenium::WebDriver.for(
    :remote,
    url: 'http://localhost:4444/wd/hub',
    desired_capabilities: :any,
  )
end

def teardown
  @driver.quit
end

def run
  setup
  yield
  teardown
end

run do
  @driver.get 'https://www.ruby-lang.org/en/'
  expect(@driver.title).to eq('Ruby Programming Language')
end

File: se_test.py
from selenium import webdriver

driver = webdriver.Remote(
  command_executor = "http://52.210.77.132:4444/wd/hub",
  desired_capabilities = {
    "browserName": "firefox",
    #"platform": "LINUX",
  }
)

try:
  driver.implicitly_wait(30)
  driver.get("http://www.python.org")
  print(driver.title)
  assert "Python" in driver.title
finally:
  driver.quit()
